### ISSUE TITLE
Enhancement: recognize `DEBIAN_FRONTEND=noninteractive` being set in the environment

### ISSUE THREAD

**Issue by cooljeanius** [fink/fink#264]
Many Debian-based CI systems set the environment variable `DEBIAN_FRONTEND` to `noninteractive` so that the CI can just run the package manager without the user having to do anything. Since Fink shares a lineage with Debian's packaging system, it would be nice if it could support this environment variable, too, for consistency. Yes, there's the `-y` flag, but on Debian they also have the `-y` flag and still support the environment variable separately. Also, sometimes there are instances where one can edit the environment in which a prompt is run, but not the exact flags passed to it.

**Comment by dmacks** [comment]
Makes sense. So just to clarify, `env("DEBIAN_FRONTEND")=="noninteractive"` should be treated as synonymous to `-y` ?

**Comment by TheSin-** [comment]
this is a bad idea, as this would break debconf

**Comment by cooljeanius** [comment]
> Makes sense. So just to clarify, `env("DEBIAN_FRONTEND")=="noninteractive"` should be treated as synonymous to `-y` ?

I'd have to double-check whatever the behavior is under Debian...

**Comment by TheSin-** [comment]
I know it is, I use it in my image maker and my octoprint scripts.  It's how you pick the front ends, curses, diag, etc etc
